### PR TITLE
Mention similar / related tools in a separate Markdown document

### LINKED ISSUE #118: Link to alternative similar tools in README
Since bump2version is not suitable for every use case, I'd like to propose a short blurb in the README linking to alternatives that do things differently.

The ones I know are zest.releaser, setuptools-scm, incremental, advbumpversion, invocations packaging.release, semantic.release.

### PR DESCRIPTION
**Pull request by florisla**
I think it would beneficial to help users determine if they're actually looking at the right tool for the job.

Fixes #118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,8 @@
 
 This is an interim fork of the excellent project that can be found here: https://github.com/peritus/bumpversion
 
-Unfortunately it seems like development has been stuck for some time and no activity has been seen from the
-author, to that end this project is a drop-in replacement that merges in some of the more important fixes.
-
-Hopefully we can merge back into the original bumpversion and carry on development there.
+In the future, both projects will re-merge with backwards
+compatibility as a goal.
```
Comment (ekohl) [inline README.md]: Should this link to the issue with the intention?

File: RELATED.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Similar or related tools
+
+* [bumpversion](https://pypi.org/project/bumpversion/) is the original project
+  off of which `bump2version` was forked.  We'll be merging
+  back with them at some point.
```
Comment (ekohl) [inline RELATED.md]: Link to the relevant issue?

File: RELATED.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+* [ADVbumpversion](https://github.com/andrivet/advbumpversion) is another fork.
+  It offers some features which are still work in progress here; it's
+  definitely our desire to merge back.
```
Comment (ekohl) [inline RELATED.md]: Is there an issue tracking this effort?

File: README.md (hunk 1 above)
Comment (florisla) [reply README.md]: Link added.

File: RELATED.md (hunk 3 above)
Comment (florisla) [reply RELATED.md]: Added link to #121

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**